util/MeasurePopulations.ts

```diff
@@ -43,3 +43,22 @@ export function getMeasurePopulations(measure: fhir4.Measure): MultiSelectData[]
   });
   return measurePopulations;
 }
+
+/**
+ * Extracts all population codes included in measure groups
```

Comment:
Based on Matt's [typescript guide](https://gist.github.com/mgramigna/bdc933fae2150216681c28d1f0f7aac7) (shoutout @mgramigna the king 👑 of typescript), I don't think we need to include the JSDoc param/return since it's not adding a lot of new information and can easily become outdated if we update this function in the future